config/software/ruby.rb

```diff
@@ -46,11 +46,20 @@
 dependency "libiconv"
 
 # The checksums below are for the *.tar.gz packages.
+version("2.5.3")      { source sha256: "9828d03852c37c20fa333a0264f2490f07338576734d910ee3fd538c9520846c" }
```

Comment:
Curious: Do you know why we need multiple versions (2.5.3, 2.5.1, 2.5.0) instead of just having `2.5.3`? Since we are using an embedded Ruby, isn't one version sufficient? If this to support cases when customers install extra plugins that has specific Ruby version requirements?

Reply:
No, these are just there to allow setting the embedded Ruby version to one of them. The version is still specified on line 31 above.
I just grabbed the changes from the upstream repo, which had all of these versions. Keeping in sync with upstream is actually a good thing, IMO.

Reply:
Agreed.